grazer/src/main/java/edu/asu/diging/grazer/core/domain/impl/FileMetadataImpl.java

```diff
@@ -0,0 +1,113 @@
+package edu.asu.diging.grazer.core.domain.impl;
+
+import java.util.Date;
+import java.util.List;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+import javax.persistence.Transient;
+
+import org.hibernate.validator.constraints.NotEmpty;
+import org.springframework.web.multipart.MultipartFile;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+
+import edu.asu.diging.grazer.core.domain.ITransformationFile;
+import edu.asu.diging.grazer.core.domain.IFileMetadata;
+
+@Entity
+@Table(name="tbl_files")
+public class FileMetadataImpl implements IFileMetadata
+{
+ 
+    @Id @GeneratedValue(strategy=GenerationType.IDENTITY) private int id;
+    @NotEmpty
+    private String label;
+    private String description;
+    private String uploader;
+    private Date date;
```

Comment:
Since a date is only useful when we know the timezone, let's use OffsetDateTime instead (https://docs.oracle.com/javase/8/docs/api/java/time/OffsetDateTime.html).